ghcide/session-loader/Development/IDE/Session.hs

```diff
@@ -395,8 +395,8 @@ loadSessionWithOptions SessionLoadingOptions{..} dir = do
            lfp <- flip makeRelative cfp <$> getCurrentDirectory
            logInfo logger $ T.pack ("Consulting the cradle for " <> show lfp)
 
-           when (isNothing hieYaml) $ mRunLspT lspEnv $
-             sendNotification SWindowShowMessage $ notifyUserImplicitCradle lfp
+           when (isNothing hieYaml) $
+             logWarning logger $ implicitCradleWarning lfp
```

Comment:
Is this even a *warning*? It seems like info would be plausible: this is something that users mostly won't care about (the message even says that!) but it's useful to have in the logs later if you're trying to figure out what happened.

Reply:
Agree, there is not much difference given we are handling logs but it can be info

Reply:
I've converted it to info